keras_hub/src/models/roformer_v2/roformer_v2_backbone.py

```diff
@@ -0,0 +1,214 @@
+        self,
+        vocabulary_size,
+        num_layers,
+        num_heads,
+        hidden_dim,
+        intermediate_dim,
+        head_size,
+        use_bias=False,
+        activation="relu",
+        dropout=0.1,
+        num_segments=2,
+        dtype=None,
+        max_wavelength=10000,
+        **kwargs,
+    ):
+        # === Layers ===
+        self.token_embedding = ReversibleEmbedding(
+            input_dim=vocabulary_size,
+            output_dim=hidden_dim,
+            embeddings_initializer=roformer_kernel_initializer(),
+            dtype=dtype,
+            name="token_embedding",
+        )
+        self.segment_embedding = keras.layers.Embedding(
+            input_dim=num_segments,
+            output_dim=hidden_dim,
+            embeddings_initializer=roformer_kernel_initializer(),
+            dtype=dtype,
+            name="segment_embedding",
+        )
+        self.embeddings_add = keras.layers.Add(
+            dtype=dtype,
+            name="embeddings_add",
+        )
+        self.embeddings_layer_norm = RoformerNorm(
+            epsilon=keras.backend.epsilon(),
+            dtype=dtype,
+            name="embeddings_layer_norm",
+        )
+        self.embeddings_dropout = keras.layers.Dropout(
+            dropout,
+            dtype=dtype,
+            name="embeddings_dropout",
+        )
+        self.transformer_layers = []
+        for i in range(num_layers):
+            layer = RoformerV2Encoder(
+                heads=num_heads,
+                head_size=head_size,
+                intermediate_size=intermediate_dim,
+                use_bias=use_bias,
+                max_wavelength=max_wavelength,
+                dropout=dropout,
+                activation=activation,
+                kernel_initializer=roformer_kernel_initializer(),
+                dtype=dtype,
+                name=f"transformer_layer_{i}",
+            )
+            self.transformer_layers.append(layer)
+
+        # === Functional Model ===
+        token_id_input = keras.Input(
+            shape=(None,), dtype="int32", name="token_ids"
+        )
+        segment_id_input = keras.Input(
+            shape=(None,), dtype="int32", name="segment_ids"
+        )
+        attention_mask = keras.ops.not_equal(token_id_input, 0)
+        # Embed tokens, positions, and segment ids.
+        tokens = self.token_embedding(token_id_input)
+        segments = self.segment_embedding(segment_id_input)
+        # Sum, normalize and apply dropout to embeddings.
+        x = self.embeddings_add((tokens, segments))
+        x = self.embeddings_layer_norm(x)
+        x = self.embeddings_dropout(x)
+        for transformer_layer in self.transformer_layers:
+            x = transformer_layer(x, attention_mask=attention_mask)
+        # Construct the two RoformerV2 outputs.
+        # The pool layer is mean pool
+        sequence_output = x
+        cls_token_index = 0
+        # roformer not use NSP task as pretrain task
```

Comment:
Just delete the `"pooled_output"` entirely then. It's really just a quirk of bert. See roberta as an example.